Mathlib/Algebra/Order/Module/Algebra.lean

```diff
@@ -0,0 +1,58 @@
+/-
+Copyright (c) 2023 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Algebra.Basic
+import Mathlib.Algebra.Order.Module.Defs
+
+/-!
+# Ordered algebras
+
+This file proves properties of algebras where both rings are ordered compatibly.
+-/
+
+variable {α β : Type*} [OrderedCommSemiring α]
+
+section OrderedSemiring
+variable (β)
+variable [OrderedSemiring β] [Algebra α β] [SMulPosMono α β] {a : α}
+
+lemma algebraMap_mono : Monotone (algebraMap α β) :=
```

Comment:
Please add a version with unfolded definition and make it `@[mono, gcongr]` (`mono` can accept `Monotone` lemmas while `gcongr` can't).